editor/debugger/editor_debugger_tree.cpp

```diff
@@ -146,24 +147,50 @@ void EditorDebuggerTree::_scene_tree_rmb_selected(const Vector2 &p_position, Mou
 /// |-E
 ///
 void EditorDebuggerTree::update_scene_tree(const SceneDebuggerTree *p_tree, int p_debugger) {
+	set_hide_root(false);
```

Comment:
Is there any way to avoid resetting it every time?

Reply:
That function already checks if the value is actually being changed before proceeding. Is there another problem that I'm not seeing?

Reply:
But if the root is hidden, this will show it and then it can be hidden again, resulting in 2 updates.
Unless that's not possible to happen?